tasks/updater.py

```diff
@@ -0,0 +1,198 @@
+"""
+Updater namespaced tasks
+"""
+
+
+import os
+import sys
+
+from invoke import task
+
+from .build_tags import filter_incompatible_tags, get_build_tags, get_default_build_tags
+from .go import deps
+from .utils import REPO_PATH, bin_name, get_build_flags, get_version, load_release_versions, timed
+
+BIN_PATH = os.path.join(".", "bin", "updater")
+
+
+@task
+def build(
+    ctx,
+    rebuild=False,
+    race=False,
+    build_include=None,
+    build_exclude=None,
+    major_version='7',
```

Comment:
do we intent to support agent 6 and 7 through the updater? Or can we assume 7?

Reply:
only 7, I removed the parameter